src/components/MovieTile.tsx

```diff
@@ -0,0 +1,116 @@
+import type { Movie } from "../types/movie";
+
+export interface MovieTileProps {
+  movie: Movie;
+  onEdit?: (movie: Movie) => void;
```

Comment:
What do you think about sending the id instead of the entire object?

```suggestion
  onEdit?: (id: movieId) => void;
```